luaui/Widgets/gui_info.lua

```diff
@@ -345,7 +345,7 @@ local function refreshUnitInfo()
 
 						if weaponDef.type == "BeamLaser" then
 							calculateLaserDPS(weaponDef, weaponDef.damages[0])
-						elseif weaponDef.customParams.cluster then -- Bullets that shoot other, smaller bullets
+						elseif weaponDef.customParams and weaponDef.customParams.cluster then -- Bullets that shoot other, smaller bullets
```

Comment:
Were weapon defs throwing nil errors for `customParams`? An empty table should be injected at initialization, so this nil check should not be required.